sentinel/.eslintrc

```diff
@@ -42,7 +43,8 @@
         "tests/**/*.js"
       ],
       "rules": {
-        "node/no-extraneous-require": "off"
+        "node/no-extraneous-require": "off",
+        "eqeqeq": ["error", "always", { "null": "ignore" }] // foo != null is a convenient way to check for null or undefined.
```

Comment:
As above, if you want to lobby for this change it should be done as a separate pr.

Reply:
I'll make that now.